ui/pages/notifications/notifications-list-item.tsx

```diff
@@ -44,13 +46,23 @@ export function NotificationsListItem({
         previously_read: notification.isRead,
       },
     });
+
     markNotificationAsRead([
       {
         id: notification.id,
         type: notification.type,
         isRead: notification.isRead,
       },
     ]);
+
+    if (
+      notification.type === TRIGGER_TYPES.SNAP &&
+      !hasProperty(notification.data, 'detailedView')
+    ) {
+      setNotificationTimeout(notification.id);
```

Comment:
When is a notification with a detailed view removed?

Reply:
On dismount